egs2/jkac/tts1/local/prep_segments.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
```

Comment:
We use black as formatter and flake8+hacking as linter.
Maybe CI is failed, please apply it.